astroplan/constraints.py

```diff
@@ -635,3 +692,11 @@ def observability_table(constraints, observer, targets, times=None,
     tab.meta['constraints'] = constraints
 
     return tab
+
+def _rescale_minmax(vals, min_val, max_val):
```

Comment:
@eteq - so all constraints will now have an additional option to output a float which "rates" the observability of a target on `(0, 1)`? I don't know if I understand what's going on with the maximum value here -- if the value is above the `max`, the rescaled value is `1` (not to be confused with `True`)?